docs/DataModels/DevLakeDomainLayerSchema.md

```diff
@@ -428,80 +429,80 @@ Events of pull requests.
-| `commit_sha` | varchar  | 255       |                                                                 | PK      |
-| `branch`     | varchar  | 255       |                                                                 |         |
-| `repo_url`   | varchar  | 255       |                                                                 |         |
-| `id`         | varchar  | 255       | This key is generated based on details from the original plugin | PK      |
-| `created_at` | datetime | 3         |                                                                 |         |
-| `updated_at` | datetime | 3         |                                                                 |         |
-
+| **field** | **type** | **length** | **description**                                                 | **key** |
+| :-------- | :-------- | :-------- |:----------------------------------------------------------------| :-------- |
+| `id` | varchar | 255) | This key is generated based on details from the original plugin | PK |
+| `created_at` | datetime | 3)                             | Devlake common.NoPKModel                                        |  |
+| `updated_at` | datetime | 3)                             | Devlake common.NoPKModel                                                |  |
+| `_raw_data_params` | varchar | 255)                           | Devlake common.NoPKModel.RawDataOrigin                                  |  |
+| `_raw_data_table` | varchar | 255)                           | Devlake common.NoPKModel.RawDataOrigin                                  |  |
+| `_raw_data_id` | bigint unsigned | Devlake common.NoPKModel.RawDataOrigin |                                                                 |  |
+| `_raw_data_remark` | longtext | Devlake common.NoPKModel.RawDataOrigin              |                                                                 |  |
+| `name` | varchar | 255) |                                                                 |  |
+| `result` | varchar | 100)                           | The result of this task                                         |  |
+| `status` | varchar | 100)                           | The status of this task                                         |  |
+| `type` | varchar | 100)                           | To indicate if this is a DEPLOYMENT                             |  |
+| `duration_sec` | bigint unsigned |                                | how long does this task take                                    |  |
+| `started_date` | datetime | 3)                             | when did this task start                                        |  |
+| `finished_date` | datetime | 3)                             | when did this task finish                                       |  |
+| `environment` | varchar | 255)                           | To indicate if this is PRODUCTION(dora)                         |  |

```

Comment:
Can we change it to `To indicate the environment in which the task is running`?

Reply:
> To indicate the environment in which the task is running

fixed